doc/rados/operations/pools.rst

```diff
@@ -213,103 +213,101 @@ For example:
+consider deleting those rules.
 
 .. prompt:: bash $
 
    ceph osd pool get {pool-name} crush_rule
 
-If the rule was "123", for example, you can check the other pools like so:
+For example, if the custom rule is "123", you can check all other pools to see whether they use the rule by running the following command:
 
 .. prompt:: bash $
 
-	ceph osd dump | grep "^pool" | grep "crush_rule 123"
+    ceph osd dump | grep "^pool" | grep "crush_rule 123"
 
-If no other pools use that custom rule, then it's safe to delete that
+If no other pools use this custom rule, then it is safe to delete the
 rule from the cluster.
 
-If you created users with permissions strictly for a pool that no longer
-exists, you should consider deleting those users too:
-
+Similarly, if you created users with permissions restricted to a pool that no longer
+exists, consider deleting those users by running commands of the following forms:
 
 .. prompt:: bash $
 
-	ceph auth ls | grep -C 5 {pool-name}
-	ceph auth del {user}
+    ceph auth ls | grep -C 5 {pool-name}
+    ceph auth del {user}
 
 
-Rename a Pool
-=============
+Renaming a Pool
+===============
 
-To rename a pool, execute:
+To rename a pool, run a command of the following form:
 
 .. prompt:: bash $
 
    ceph osd pool rename {current-pool-name} {new-pool-name}
 
-If you rename a pool and you have per-pool capabilities for an authenticated
-user, you must update the user's capabilities (i.e., caps) with the new pool
-name.
+If you rename a pool for which an authenticated user has per-pool capabilities,
+you must update the user's capabilities ("caps") to refer to the new pool name.
+
 
-Show Pool Statistics
-====================
+Showing Pool Statistics
+=======================
 
-To show a pool's utilization statistics, execute:
+To show a pool's utilization statistics, run the following command:
 
 .. prompt:: bash $
 
    rados df
 
-Additionally, to obtain I/O information for a specific pool or all, execute:
+To obtain I/O information for a specific pool or for all pools, run a command
+of the following form:
 
 .. prompt:: bash $
 
    ceph osd pool stats [{pool-name}]
 
 
-Make a Snapshot of a Pool
-=========================
+Making a Snapshot of a Pool
+===========================
 
-To make a snapshot of a pool, execute:
+To make a snapshot of a pool, run a command of the following form:
 
 .. prompt:: bash $
 
    ceph osd pool mksnap {pool-name} {snap-name}
 
-Remove a Snapshot of a Pool
-===========================
+Removing a Snapshot of a Pool
+=============================
 
-To remove a snapshot of a pool, execute:
+To remove a snapshot of a pool, run a command of the following form:
 
 .. prompt:: bash $
 
    ceph osd pool rmsnap {pool-name} {snap-name}
 
 .. _setpoolvalues:
 
+Setting Pool Values
+===================
 
-Set Pool Values
-===============
-
-To set a value to a pool, execute the following:
+To set pool values for keys, run a command of the following form:
```

Comment:
To assign values to a pool's configuration keys, run

Reply:
Accepted.